src/models/registry.ts

```diff
@@ -0,0 +1,9 @@
+export class Resgistry {
```

Comment:
Typo

Reply:
I've looked into the stringify method and it seems it doesn't work properly with classes but there's a way to implement it, like this: 
http://codebuckets.com/2018/06/17/extending-typescript-to-serialise-map-objects-to-json/
What do you think?